content/docs/command-reference/checkout.md

```diff
@@ -10,8 +10,8 @@ usage: dvc checkout [-h] [-q | -v] [--summary] [-d] [-R] [-f]
                     [--relink] [targets [targets ...]]
 
 positional arguments:
-  targets          DVC-files to checkout. Optional. (Finds all
-                   DVC-files in the workspace by default.)
+  targets          DVC-files or files/directories to checkout. Optional. (Finds
+                   all DVC-files in the workspace by default.)
```

Comment:
We probably need to switch to `Stage/file/directory to checkout` terminology instead. :thinking:

Reply:
This is sloppy :(

Reply:
Sounds like this could be an issue to open in the core repo, which should be reflected in docs when addressed 🙂

Reply:
Actually this has been standardize in the cmd ref to:

```md
  targets        Limit command scope to these stages or .dvc files.
                 Using -R, directories to search for stages or .dvc
                 files can also be given.
```

Reply:
@efiop actually... Are these all wrong? Should it be "stages or tracked files/dirs" now? Checking...

Reply:
Hm, nope. This is correct. .dvc files or stage names only (checked status only though). So how can you be granular? 🤔

Reply:
Hmmm. I see `status -c` does accept tracked file names. I don't get why not on local mode (I get `ERROR: failed to obtain data status - 'dvc.yaml' does not exist.`) but OK, I'll document this for now... ⏳

Reply:
Updated. Could you please review the changes to `status` when you have a chance, @efiop ? Thanks

Reply:
Actually, status, push, and pull are all ready for review 🙂